### PR TITLE
Add task_run_costs table to fxci custom namespace

### PR DESCRIPTION
**Pull request by ahal**

### CODE REVIEW

File: custom-namespaces.yaml [hunk 1]
```diff
@@ -1696,6 +1696,10 @@ fxci:
       type: table_explore
       views:
         base_view: task_runs
+    task_run_costs:
```
Comment (fbertsch) [inline custom-namespaces.yaml:1699]: Do you want to add measures here?

**Review by fbertsch — APPROVED** [review]
LGTM

File: custom-namespaces.yaml (hunk 1 above)
Comment (ahal) [reply custom-namespaces.yaml:1707]: I'm not sure, I don't think so? But I don't really know what I'm doing :)

Looking at the other measures defined in this file, I only see count and sum. In this case, cost is already the total for the task run. So it doesn't make sense to sum or count it *within this table explore*.

That said once this lands, I'm planning to write some explores in the spoke project that allow joining this table to the other tables in this namespace. In those cases we'll want to be able to sum cost across tasks, filtering on various attributes of tasks. Will accomplishing that require a measure defined here?

Comment (fbertsch) [reply custom-namespaces.yaml:1707]: If you're doing further refinement of these generated explores, no need to define things here. Carry on, I'll merge this as-is

**Pull request merged**